hashing/src/lib.rs

```diff
@@ -1,174 +1,165 @@
-//! Cryptographic hash type and function.
+#![cfg_attr(not(feature = "std"), no_std)]
 
-// TODO - remove once schemars stops causing warning.
-#![allow(clippy::field_reassign_with_default)]
+/// The number of bytes in a Blake2b hash
```

Comment:
That probably belongs to line 42?
And maybe

```rust
/// The number of bytes in a digest
```

would be more accurate: blake2b allows different sizes; 32 bytes is our choice.